Updating wording for contacting support (#1045)
codatio/codat-docs: docs/integrations/accounting/zoho-books/zoho-book-limits.md

| Customers         | Address and contact information |
| Invoices          | Line items                      |

> To remove these restrictions and pull complete datasets for the supported Zoho Books data types, please raise a ticket with our support team through [this form](https://codat.zendesk.com/hc/en-gb/requests/new). **After removing the restrictions, fetching large volumes of data from the above data types can take multiple days to complete.**

Reviewer: ```suggestion
To remove these restrictions and pull complete datasets for the supported Zoho Books data types, please raise a ticket through our [support request form](https://codat.zendesk.com/hc/en-gb/requests/new). **After removing the restrictions, fetching large volumes of data from the above data types can take multiple days to complete.**
```
